### PR TITLE
Command to extend sector expiration

### LINKED ISSUE #5011: Expose a CMD/API to extend sector lifetime
This is already implemented on the spec-actor, we just need to expose it in Lotus to call `ExtendSectorExpiration`.

`lotus-miner sectors extend-sector <sector id> <term>`

### PR DESCRIPTION
**Pull request by magik6k**
Resolves #5011

### CODE REVIEW

**Comment by magik6k** [comment]
(tested manually on my miner)

File: cmd/lotus-storage-miner/sectors.go [hunk 1]
```diff
@@ -403,6 +407,100 @@ var sectorsRefsCmd = &cli.Command{
...
+		}
+
+		params := &miner0.ExtendSectorExpirationParams{}
+		for l, numbers := range sectors {
+
+			params.Extensions = append(params.Extensions, miner0.ExpirationExtension{
+				Deadline:      l.Deadline,
+				Partition:     l.Partition,
+				Sectors:       bitfield.NewFromSet(numbers),
+				NewExpiration: abi.ChainEpoch(cctx.Int64("new-expiration")),
```
Comment (arajasek) [inline cmd/lotus-storage-miner/sectors.go:472]: Would be nice to sanity-check this param (positive / greater than all the current expiration epochs / whatever actors code expects)

**Review by arajasek — APPROVED** [review]
Approved with an optional suggestion

File: cmd/lotus-storage-miner/sectors.go (hunk 1 above)
Comment (magik6k) [reply cmd/lotus-storage-miner/sectors.go:472]: Pretty sure that MpoolPush will fail with a nice error message when that happens

**Pull request merged**